pkg/managed/daemonsets_test.go

```diff
@@ -32,11 +32,11 @@ func TestCreateDaemonSetsVmiSystem(t *testing.T) {
 
 	VzSystemInfo.SynBinding.Name = constants.VmiSystemBindingName
 
-	err := CreateDaemonSets(VzSystemInfo, clusterConnections, verrazzanoURI)
+	err := CreateDaemonSets(VzSystemInfo, clusterConnections, verrazzanoURI, "docker://19.3.11")
```

Comment:
Are there any unit tests for the `containerd://` case?

Reply:
good point, I will add.

Reply:
Addressed in commit e0cb7e9523fa6287222cc134a03a69a0cfd26fa2